src/EventBus.js

```diff
@@ -1,100 +1,413 @@
+/**
+ * Created by bona on 2016/12/12.
+ */
```

Comment:
I'm sorry. The first time I found the EventBus library, I copied the code directly and pasted it into the project file created by IDEA without noticing the copyright information. Later, it was necessary to give feedback on the changes I made, the code will be submitted directly to the git.